target/linux/ath79/dts/ar7242_meraki_mr12.dts

```diff
@@ -0,0 +1,186 @@
+// SPDX-License-Identifier: GPL-2.0-or-later OR MIT
+/dts-v1/;
```

Comment:
this can be dropped.

Reply:
Done.

Reply:
I think he meant only the DTS tag to be dropped

we still keep the license line I'm pretty sure...